riotfile.py

```diff
@@ -486,5 +486,11 @@ def select_pys(min_version=min(SUPPORTED_PYTHON_VERSIONS), max_version=max(SUPPO
                 ),
             ],
         ),
+        Venv(
+            name="urllib3",
+            pys=SUPPORTED_PYTHON_VERSIONS,
+            pkgs={"urllib3": [">=1.22,<1.27"]},
```

Comment:
We should also test with the latest version of urllib3
```suggestion
            pkgs={"urllib3": [">=1.22,<1.27", latest]},
```